core/modules/system/system.theme.inc

```diff
@@ -178,6 +178,9 @@ function theme_system_modules_fieldset($variables) {
     if ($required_by) {
       $requirements .= '<div class="admin-requirements">' . t('Required by:') . ' ' . implode(', ', $module['#required_by']) . '</div>';
     }
+    if (isset($module['#project'])) {
+      $requirements .= '<div class="admin-requirements project-links">' . t('Links:') . ' ' . l(t("BackdropCMS.org"), 'https://backdropcms.org/project/' . $key) . ', ' . l(t("GitHub"), 'https://github.com/backdrop-contrib/' . $key) . '</div>';
```

Comment:
Because projects can contain multiple modules, we should use the `$module['#project']` variable here. This will result in multiple modules pointing at the same GitHub URL, but that is fine.

```suggestion
      $requirements .= '<div class="admin-requirements project-links">' . t('Links:') . ' ' . l(t("BackdropCMS.org"), 'https://backdropcms.org/project/' . $module['#project']) . ', ' . l(t("GitHub"), 'https://github.com/backdrop-contrib/' . $module['#project']) . '</div>';
```